quipucords/fingerprinter/task.py

```diff
@@ -87,12 +88,15 @@
 
 # keys are in reverse order of accuracy (last most accurate)
 # (date_key, date_pattern)
-RAW_DATE_KEYS = \
-    [('date_yum_history', ['%Y-%m-%d']),
-     ('date_filesystem_create', ['%Y-%m-%d']),
-     ('date_anaconda_log', ['%Y-%m-%d']),
-     ('registration_time', ['%Y-%m-%d %H:%M:%S', '%Y-%m-%d %H:%M:%S %z']),
-     ('date_machine_id', ['%Y-%m-%d'])]
+RAW_DATE_KEYS = OrderedDict(
+    [
```

Comment:
dict being currently ordered is more a happy accident than  a feature we should count on. That might change in the future. If you rely on order, use `OrderedDict`.

Reply:
Mmm what I said isn't true anymore. When this order came it wasn't something to be relied upon
https://docs.python.org/3/whatsnew/3.6.html#new-dict-implementation

I didn't knew that this got "oficial" status on 3.7.
> Changed in version 3.7: Dictionary order is guaranteed to be insertion order. This behavior was an implementation detail of CPython from 3.6.
https://docs.python.org/3/library/stdtypes.html#mapping-types-dict

Since we don't rely on any of the other features from `OrderedDict` we can use `dict`, then.

Reply:
Let's go with dict then.